.circleci/config.yml

```diff
@@ -72,7 +72,28 @@ jobs:
             export PR_NUMBER=$CIRCLE_PR_NUMBER
             << parameters.command >>
 
+  sonarqube:
+    docker:
+      - image: checkstyle/jdk-11-groovy-git-mvn:11.0.13__3.0.9__2.25.1__3.6.3
```

Comment:
> groovy
> 3.0.9

https://github.com/checkstyle/contribution/tree/master/checkstyle-tester#setup
> groovy (2.4.8) is required to execute

https://github.com/checkstyle/checkstyle/pull/12432#issuecomment-1322468461
> which version ci using ?

Just pointing out I think we need to make these all consistent otherwise it will get confusing.

Reply:
Yes, I have been thinking about having one image for all CI, since we do so many similar things